api/controller.py

```diff
@@ -625,9 +626,20 @@ def load_lane(self, lane_identifier):
             elif isinstance(lane, WorkList):
                 lane.children = [self._db.merge(child) for child in lane.children]
         else:
-            lane = get_one(
-                self._db, Lane, id=lane_identifier, library_id=library_id
-            )
+            try:
+                lane_identifier = int(lane_identifier)
+            except ValueError, e:
+                pass
```

Comment:
Previously if you asked for a lane with a string name (rather than a numeric ID) you would get a database error.